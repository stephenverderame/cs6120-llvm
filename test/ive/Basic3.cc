// NOLINTBEGIN
#include <cstdio>
int main()
{
    int m = 20;
    for (int i = 0; i < 500; i += m) {
        printf("%d\n", i);
    }
}
// NOLINTEND