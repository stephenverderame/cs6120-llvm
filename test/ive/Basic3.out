Basic IV:   %3 = phi i32 [ 0, %0 ], [ %5, %2 ]
Basic IV:   %5 = add nuw nsw i32 %3, 20
